### PR TITLE
Fixes of autotests C353639, C353640 and 358140

### PR DESCRIPTION
**Pull request by Sherzod-Kenjaev**
https://foliotest.testrail.io/index.php?/cases/view/353639
https://foliotest.testrail.io/index.php?/cases/view/353640
https://foliotest.testrail.io/index.php?/cases/view/358140

### CODE REVIEW

**Comment by Sherzod-Kenjaev** [comment]
> @Sherzod-Kenjaev why have you closed the previous pr?

@penfriend sorry for that, I had problem in getting CLA license signing and the reason is that I did commit and push my changes using my other github account which doesn't have access to folio, I forgot to change my git config email to my epam email, so I was not able to merge it. Discussed with Oll and created new PR then pushed my last update.

**Comment by penfriend** [comment]
Got you! No worries. Actually to sign in the license you could add one more email to your account and then the issue with CLA license would be solved.  Before merging the pull request you need to get two approvals from reviewers and all the comments need to be resolved.

**Review by TetianaParanich — APPROVED** [review]

**Review by OllIsanbaev — APPROVED** [review]

**Review by OllIsanbaev — APPROVED** [review]

**Pull request merged**
